projects/packages/assets/src/class-assets.php

```diff
@@ -307,12 +319,13 @@ public static function normalize_path( $path ) {
 	 * This wrapper handles all of that.
 	 *
 	 * @since 1.12.0
+	 * @since $$next-version$$ Add a new `strategy` option to leverage >= 6.3. script strategy feature. The `async` option is deprecated.
```

Comment:
typo, and clarify what "6.3" refers to.
```suggestion
	 * @since $$next-version$$ Add a new `strategy` option to leverage WP >= 6.3 script strategy feature. The `async` option is deprecated.
```